packages/ui-components/style/icons/filetype/console.svg

```diff
@@ -1,4 +1,6 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="16" viewBox="0 0 200 200">
+<svg aria-labelledby="alt" xmlns="http://www.w3.org/2000/svg" width="16" viewBox="0 0 200 200">
+<alt id="console icon" lang="en">Console icon </alt>
+<desc id="console icon">Blue square background with white greater than sign and a underscore</desc>
```

Comment:
We cannot do this unfortunately. The IDs have to be unique. Each icon is used multiple times in JupyterLab so the descriptions cannot be embedded in SVG files like that without further modification of icon code.

Reply:
This is something that one would need to figure out. Three potential solutions, in order from the worst to the best:
1. (worst) [this answer](https://stackoverflow.com/a/52269819) explains the SVG injector pattern where code responsible for inlining SVGs (`LabIcon` in our codebase) adds a unique suffix to SVGs on each injection (which however would need to benchmarked as this could lead to performance deterioration as more intelligent browsers would not be able to deduplicate DOM nodes as those would not longer be identical).
2. (ok short term solution) the SVG injector could instead only insert the `alt`/`desc` tag once in a dedicated place in DOM (only inser it if an element with such an ID exists) and remove the element from SVG strings otherwise (this is close to what @fcollonval proposed in his review first highlighting the problem).
3. (best, but potentially limiting customisation) rewrite out SVG injector to use `<use>` with a single `<title>` (see https://css-tricks.com/accessible-svgs/#aa-icons)

As this is rather a more involved task and a code design decision that would need to be discussed separately.

Further the code as proposed does not work. In `aria-labelledby="X"` the `X` should point to an ID of another element. Here `aria-labelledby="alt"` points to nothing since there is no element with `id="alt"`.

Taking a step back, there should be no fundamental reason to use [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) in the first place (there was one once due to bugs in screen readers/browsers). Instead one would use `<title>`. Also, using `aria-label` would solve the IDs issue too.

Reply:
I opened https://github.com/jupyterlab/jupyterlab/pull/14320 to track this. For now, I would suggest to either:
- (preferable) revert all changes on SVG files (until https://github.com/jupyterlab/jupyterlab/pull/14320 is solved), or
- (problematic for non-English users) use `<title>`/`<desc>` without `aria-labelledby` and without adding any IDs